fetch_model.py

```diff
@@ -0,0 +1,16 @@
+import os
+import ellipsis as el
+
+def fetch_model():
+    """Downloads pretrained model from remote drive."""
+
+    # Ensure the directory exists
+    directory = 'model'
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+    
+    # Download pretrained model
+    el.path.file.download(pathId='c7894e6f-1a2f-40b2-ab48-01e01e915790', filePath=directory+'\\model.joblib')
```

Comment:
```suggestion
    el.path.file.download(pathId='c7894e6f-1a2f-40b2-ab48-01e01e915790', filePath=directory+'/model.joblib')
```

Reply:
I don't know if this works for you, but for it to download on my machine I need to do this